src/h5web/visualizations/nexus/utils.ts

```diff
@@ -54,19 +59,15 @@ export function getNxAxisNames(group: HDF5Group): (string | undefined)[] {
   return axisNames.map((a) => (a !== '.' ? a : undefined));
 }
 
-export function getNxDataGroup(
-  entity: HDF5Entity | undefined,
+export function findNxDataGroup(
+  group: HDF5Group,
   metadata: HDF5Metadata
 ): HDF5Group | undefined {
-  if (!entity || !isGroup(entity)) {
-    return undefined;
```

Comment:
Since we already know that the first argument is a group (because this has been checked/asserted before calling `findNxDataGroup`), we can remove this check.